tests/Http/CookieTest.php

```diff
@@ -40,6 +40,23 @@ public function testRemove()
         $this->assertFalse(Cookie::remove('none'));
     }
 
+    /**
+     * we have to manually test for 32-bit servers
+     * and used set() method instead forever()
+     *
+     * PHP_INT_MAX values
+     * - 32-bit 2147483647
+     * - 64-bit 253402214400
+     */
+    public function testForeverOn32BitServer()
+    {
+        // to be tested manually by sending the max integer on a 32-bit server
+        Cookie::set('forever', 'bar', ['lifetime' => 2147483647]);
+
+        $this->assertSame('703a07dc4edca348cb92d9fcb7da1b3931de0a85+bar', $_COOKIE['forever']);
+        $this->assertTrue(Cookie::exists('forever'));
+    }
```

Comment:
You know, it's not actually a 32-bit test 🙈 Let's remove it if you wish.

Reply:
Hm, good point. 32-bit is not really testable without a 32-bit PHP binary. But I'd say it's fine as the year 2038 is coming closer and closer. 😜
Just kidding: It's an edge-case and we have now verified it manually and fixed it. So we don't really need a test if it's not viable (which it isn't).